chain/params.go

```diff
@@ -21,6 +21,7 @@ type Params struct {
 	BlockGasTarget uint64                 `json:"blockGasTarget"`
 
 	// Access control configuration
+	SuperAdminAllowBlock      *types.Address     `json:"superAdminAllowBlock,omitempty"`
```

Comment:
Can we call it `SuperAdminAllowLists` ?

Reply:
Probably `SuperAdminAccessLists` suits the best, as it is applicable to both allow and block lists.

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1839/commits/0387a8b41875f716ef490c2f24f2b2e2ad086266